core/src/main/scala/org/ensime/core/Analyzer.scala

```diff
@@ -51,17 +52,22 @@ class Analyzer(
     broadcaster: ActorRef,
     indexer: ActorRef,
     search: SearchService,
+    scoped: List[EnsimeProjectId],
```

Comment:
I wonder if perhaps we shouldn't just create a version of the `EnsimeConfig` with the `projects` filtered to include only the scoped projects. It would reduce the risk of accidental typechecking of something that we haven't been scoped to see. Principle of least power, and all.